Basic-Car-Maintenance/Shared/Dashboard/Views/DashboardView.swift

```diff
@@ -20,6 +20,12 @@ struct DashboardView: View {
         viewModel = DashboardViewModel(authenticationViewModel: authenticationViewModel)
     }
     
+    private var eventDateFormat: DateFormatter = {
+            let formatter = DateFormatter()
+            formatter.dateStyle = .short
+            return formatter
+        }()
```

Comment:
The indentation is off, the body of the computed property is indented too far, and the ending brace `}` should be even with the word `private`
I'll fix this manually, don't worry about it